docs/api_call.md

```diff
@@ -1,1352 +1,1154 @@
 
-    - *CURL command:*
+   For example, bot might ask for departure date to book a flight and user might reply with a date. Now, it is difficult to disambiguate whether its departure date or arrival date unless, we know what bot is asking for?
 
-      ```shell
-      URL='localhost'
-      PORT=8081
-      ```
+   ```
+   bot: Please help me with date of departure?
+   user: 23rd March
+   ```
 
-      ```shell
-      curl -i 'http://'$URL':'$PORT'/v1/text/?message=i%20wanted%20to%20watch%20movie&entity_name=movie&structured_value=inferno&fallback_value=None&bot_message='
-      ```
+### **Output Format**
 
-    - *CURL Output*:
+The output of entities detected will be stored in a list of dictionary containing the following structure:
 
-    - ​
+```
+[
+    {
+        "entity_value": entity_value,
+        "detection": detection_method,
+        "original_text": original_text
+    },
+]
+```
 
-      ```json
-      {
-        "data": [
-          {
-            "detection": "structure_value_verified",
-            "original_text": "inferno",
-            "entity_value": {
-              "value": "Inferno"
-            }
-          }
-        ]
-      }
-      ```
+Consider the following example for detailed explanation:
 
-  - ​
+```
+"I want to order two litre pepsi."
+```
 
+- entity_value: This will store the value of entity (i.e entity value) that is detected. The output will be in dictionary format. For example, `{"value": "2", "unit": "quantity"}`.
+- detection: This will store how the entity is detected i.e. whether from *message*, *structured value* or *fallback value*.
+- original_text: This will store the actual value that is detected. For example, `two`.
 
-### phone_number
+```json
+Example 1:
+input:  message = 'I want to order 2 burgers from mainland china at 3 pm '
+        entity_name = 'restaurant'
+        structured_value = None
+        structured_value_verification = 0
+        fallback_value = None
+        bot_message = None
 
-- This functionality calls the PhoneDetector class to detect textual entities.
+output:
+[
+  {
+    "detection": "message",
+    "original_text": "2",
+    "entity_value": { "value": "2", "unit": null},
+    "language": "en"
+  },
+  {
+    "detection": "message",
+    "original_text": "3 pm",
+    "entity_value": { "mm": 0, "hh": 3, "nn": "pm"},
+    "language": "en"
+  }
+]
 
-- Example:
+Example 2:
+// referenced date is 19th feb
+input:  message = 'I have my maths exam next Saturday.'
+        entity_name = 'date'
+        structured_value = None
+        structured_value_verification = 0
+        fallback_value = None
+        bot_message = None
 
-  - Example 1: 
+output:[
+{
+    "detection": "message",
+    "original_text": "inferno",
+    "entity_value": {"value": {"mm": 03, "yy": 2019, "dd": 02, "type": "date"}
+  }, 
```

Comment:
is the output part meant to be a valid JSON ?
JSON does not support trailing commas

Reply:
Considering v2 will be called, shouldn't there be a `"language": "en"` in output and if DateAdvanced will be called then the output format is incorrect